### ISSUE TITLE
Installing from github can create a C++/JS mismatch because of prebuilds

### ISSUE THREAD

**Issue by dragonfire535** [Automattic/node-canvas#1250]
## Issue or Feature

Trying to call `loadImage` results in a TypeError (run with a Discord bot's eval command):
```js
TypeError: Cannot read property 'call' of undefined
     at setSource (/root/xiao/node_modules/canvas/lib/image.js:102:14)
     at Image.set (/root/xiao/node_modules/canvas/lib/image.js:65:9)
     at Promise (/root/xiao/node_modules/canvas/index.js:34:15)
     at new Promise (<anonymous>)
     at loadImage (/root/xiao/node_modules/canvas/index.js:23:10)
     at eval (eval at run (/root/xiao/node_modules/discord.js-commando/src/commands/util/eval.js:57:22), <anonymous>:2:1)
     at EvalCommand.run (/root/xiao/node_modules/discord.js-commando/src/commands/util/eval.js:57:22)
     at CommandoMessage.run (/root/xiao/node_modules/discord.js-commando/src/extensions/message.js:214:34)
```
This does not happen until 1046abfc7ced2a32bdbcee70d87f52f56b20277a, using 5301420f0cd7f65ee60be33a1ec4d9d9fe557cd2 works as expected.

## Steps to Reproduce

```js
const { loadImage } = require('canvas');
loadImage(<Buffer or path>);
```

## Your Environment
* Version of node-canvas (e.g. 1.4.0): 1046abfc7ced2a32bdbcee70d87f52f56b20277a
* Environment (e.g. node 4.2.0 on Mac OS X 10.8): Ubuntu 18.04.1

**Comment by zbjornson** [comment]
I can't reproduce this using a local file path or a Buffer, and there's heavy usage of `loadImage` in the test suite. Do you have more info?

**Comment by dragonfire535** [comment]
https://github.com/dragonfire535/xiao/tree/master/commands/image-edit none of the commands in this collection here that use `loadImage` work, all throw the error seen above on the latest commit.

**Comment by zbjornson** [comment]
No release has been made with 1046abf yet, so I assume you're installing from github? Did you rebuild the module?

**Comment by dragonfire535** [comment]
I am installing from github, yes.

**Comment by zbjornson** [comment]
^ Did you rebuild after pulling? There was a change in the C++ code in that commit, which requires rebuilding the module.

**Comment by dragonfire535** [comment]
It works when providing `--build-from-source` to the install, so that's likely the reason. The prebuild, however, is currently broken.

**Comment by zbjornson** [comment]
Ah, node-pre-gyp will fetch the latest prebuild, which in this case creates a mismatch between the JS and C++. That should be fixable from the `install` script, but you found the workaround (`--build-from-source`).

(Heads-up, I'm going to change this issue's title to reflect that.)

**Comment by LinusU** [comment]
Should be fixed in `2.2.0` 🚀
